packages/runtime/src/components/builtin/Carousel/Carousel.tsx

```diff
@@ -72,19 +72,18 @@ type Props = {
   delay?: NumberValue
   showArrows?: CheckboxValue
   arrowPosition?: ResponsiveIconRadioGroupValue<'inside' | 'center' | 'outside'>
-  arrowColor?: ResponsiveColorValue
-  arrowBackground?: ResponsiveColorValue
+  arrowColor?: ResponsiveColor
+  arrowBackground?: ResponsiveColor
   showDots?: CheckboxValue
-  dotColor?: ResponsiveColorValue
+  dotColor?: ResponsiveColor
   slideBorder?: BorderValue
   slideBorderRadius?: BorderRadiusValue
 }
 
-const Wrapper = styled.div<{ width: Props['width']; margin: Props['margin'] }>`
+const Wrapper = styled.div<{ margin: Props['margin'] }>`
   position: relative;
   display: flex;
   flex-direction: column;
-  ${cssWidth('400px')}
```

Comment:
Where did the defaulted `400px` value go?

Reply:
https://github.com/makeswift/makeswift/pull/39#discussion_r852432673